templates/kubernetes/what-is-kubernetes.html

```diff
@@ -644,26 +642,13 @@ <h4 class="p-heading-icon__title">Whitepaper</h4>
         </div>
         <h3 class="p-heading--four">
           <a class="p-link--external" href="https://pages.ubuntu.com/container-whitepaper.html">
```

Comment:
Sorry I missed this, this link should be: 

```suggestion
          <a href="/engage/kubernetes-deployment-enterprise-whitepaper">
```